notebooks/lesson14_image-dataset.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>maybe just add something like "Every class (i.e. image label) ..." just because they might get confused by the phrase "helper class" later on</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>